src/plugins/intel_cpu/src/emitters/x64/jit_load_store_emitters.cpp

```diff
@@ -477,8 +477,10 @@ void jit_load_emitter::load_words_to_dword_extension(const Vmm &vmm, const Xbyak
     bool is_f16 = (prc == ov::element::f16);
     bool is_signed = prc.is_signed();
 
-    if (is_f16 && !mayiuse(cpu::x64::avx512_core_fp16))
-        OPENVINO_THROW("Load emitter in ", name_, " only support fp16 on platform with avx512_core_fp16.");
+    if (is_f16 && !mayiuse(cpu::x64::avx512_core_fp16) && !mayiuse(cpu::x64::avx2_vnni_2))
+        OPENVINO_THROW("Load emitter in ",
+                       name_,
+                       " only support fp16 on platform with avx512_core_fp16 or avx2_vnni_2.");
```

Comment:
the same as in the store case, `mayiuse(cpu::x64::avx512_core) && mayiuse(cpu::x64::avx2)` should be enough

Reply:
done